src/Components/CreateImageWizard/steps/ImageOutput/TargetEnvironment.tsx

```diff
@@ -353,9 +349,48 @@ const TargetEnvironment = () => {
             data-testid="checkbox-image-installer"
           />
         )}
-        {supportedEnvironments?.includes('wsl') && wslFlag && (
+        {supportedEnvironments?.includes('wsl') && (
           <Checkbox
-            label="WSL - Windows Subsystem for Linux (.tar.gz)"
+            label={
+              <>
+                WSL - Windows Subsystem for Linux (.tar.gz)
+                <Popover
+                  maxWidth="30rem"
+                  position="right"
+                  bodyContent={
+                    <TextContent>
+                      <Text component={TextVariants.h5}>
+                        WSL is not officially supported by Red Hat.
+                      </Text>
+                      <Text>
+                        Unfortunately Windows Subsystem for Linux is currently not supported by Red Hat.
```

Comment:
Looks great! Tiny nitpick, I'd personally remove "unfortunately" to avoid implying a stance towards Red Hat not supporting WSL and to keep things neutral.

But if the text already went through someone from copy or UX then I'm happy to merge as is.

Reply:
This was Shayna's suggestion (see the Figma link in the Epic), but I agree, let's get some more eyes on the actual text.